src/main/java/gui/BlurHelper.java

```diff
@@ -0,0 +1,128 @@
+package gui;
+
+import javafx.geometry.Bounds;
+import javafx.geometry.Point2D;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.Node;
+import javafx.scene.SnapshotParameters;
+import javafx.scene.effect.GaussianBlur;
+import javafx.scene.image.ImageView;
+import javafx.scene.image.WritableImage;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Created by markv on 5/3/2016.
+ * Class to assist in quickly blurring behind a supplied node.
+ * Also has support for moving elements!
+ */
+public class BlurHelper {
+
+    private WritableImage writableImage;
+    private SnapshotParameters parameters;
+    private Bounds bounds;
+
+    @Getter
+    private GaussianBlur gaussianBlur;
```

Comment:
One line is enough, just for documentation purposes. More comments = more better